genie/input_to_database.py

```diff
@@ -868,8 +884,13 @@ def center_input_to_database(
 
     center_input_synid = genie_config["center_config"][center]["inputSynId"]
     logger.info("Center: " + center)
+    # Don't download all the files when only validate
     center_files = extract.get_center_input_files(
-        syn, center_input_synid, center, process
+        syn=syn,
+        synid=center_input_synid,
+        center=center,
+        process=process,
+        downloadFile=(not only_validate),
```

Comment:
So we actually don't want to download all the files when we validate the files, because there is validation logic to check if we need to actually validate the files.

1. If the entity name is changed
2. If the file md5 is changed

Against what is on the validation status table.

Reply:
One thing to note here is that if someone only uploads the sample file, it's not going to download the patient file...

That does raise the complexity of the code a bit....

Reply:
i agree, let's comment it out first and handle it in another ticket